marketplace.json

```diff
@@ -5,14 +5,17 @@
       "image_uri": "https://files.readme.io/39894f8-small-Monte_Carlo_Logo_Primary.png",
       "tagline": "This Block provides access to the various metadata Monte Carlo collects to support a range of analytical and tracking use cases. Many of our customers today access this data to define and track SLI performance, monitor data asset usage, determine the most important (or least important) data assets, and more."
     },
-  
+
     "constants": {
       "CONNECTION_NAME": {
         "label": "Looker Connection Name",
         "value_constraint": "connection"
       },
       "DATASET_NAME": {
         "label": "Monte Carlo Dataset Name"
+      },
+      "DATASET_NAME": {
```

Comment:
Should this be `TABLE_PREFIX`?

Reply:
Amazing! Found a bug in something you didn't even know! I fixed it.